pkgs/development/python-modules/root/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  buildPythonPackage,
+  python,
+  pkgs,
+}:
+
+buildPythonPackage rec {
+  # ROOT builds the C++ libraries and CPython extensions in one package. We
+  # therefore need to override the root package to build against the requested
+  # Python version.
+  unwrapped = pkgs.root.override { python3 = python; };
+
+  # Properties like pname, version, and meta are taken from the main ROOT package
+  pname = unwrapped.name;
+  version = unwrapped.version;
+
+  src = null;
+
+  format = "other"; # disables setuptools/pyproject logic
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  # Create only the parent directory of python.sitePackages, such that we can
+  # create the sitePackages directory as a symlink to the ROOT library
+  # directory.
+  installPhase = ''
+    mkdir -p $out/${python.sitePackages}
+    rmdir $out/${python.sitePackages}
+    ln -s ${unwrapped}/lib $out/${python.sitePackages}
+  '';
+
+  # We can conveniently check if everything works by trying to import some C++
+  # namespaces via ROOT. Since these are looked up dynamically in the libraries
+  # via ROOTs CPython extension, these checks already cover the fragile parts
+  # of the ROOT python module.
+  pythonImportsCheck = [
+    "ROOT"
+    "ROOT.Experimental"
+    "ROOT.Math"
+    "ROOT.RooFit"
+  ];
+
+  # Do some checks to see if the ROOT C++ interpreter works.
+  checkPhase = ''
+    export PYTHONPATH=$PYTHONPATH:$out/${python.sitePackages}
+
+    # Basic C++ interpreter test with standard library class
+    cmd1='ROOT.gInterpreter.ProcessLine("std::vector<int> d{0, 1, 2, 3};")'
+    echo "Trying out: $cmd1"
+    ${python.interpreter} -c 'import ROOT; '"$cmd1"
+
+    # Make sure we can also load libraries via the interpreter, for example
+    # libCore from ROOT itself. Successful load returns status code zero.
+    cmd2='assert(ROOT.gInterpreter.Load("libCore") == 0)'
+    echo "Trying out: $cmd2"
+    ${python.interpreter} -c 'import ROOT; '"$cmd2"
```

Comment:
This sounds like it is duplicated with the above imports check

Reply:
Maybe we could load a third-party library in `passthru.tests` instead. It's also okay to keep it on hold until we managed to package `roottest`.